test/Microsoft.ML.FSharp.Tests/SmokeTests.fs

```diff
@@ -125,7 +125,7 @@ module SmokeTest1 =
             |> model.Predict
 
         let predictionResults = [ for p in predictions -> p.Sentiment ]
-        Assert.Equal<bool list>(predictionResults, [ false; true; true ])
+        Assert.Equal<Microsoft.ML.Runtime.Data.DvBool list>(predictionResults, [ Microsoft.ML.Runtime.Data.DvBool.False; Microsoft.ML.Runtime.Data.DvBool.True; Microsoft.ML.Runtime.Data.DvBool.True ])
```

Comment:
>Microsoft.ML.Runtime.Data.DvBool.False; Microsoft.ML.Runtime.Data.DvBool.True; Microsoft.ML.Runtime.Data.DvBool.True ]) [](start = 81, length = 119)

So why is this change necessary here?

Reply:
Reverted this file. Probably got mixed with the DvBool change somehow.

---
In reply to: [212060926](https://github.com/dotnet/machinelearning/pull/683#discussion_r212060926) [](ancestors = 212060926)